lib/datasets.ts

```diff
@@ -347,4 +354,17 @@ export default class StarredQueriesService extends HTTPClient {
             .then((response) => {
                 return response.data;
             });
+
+    aplQuery = (query: APLQuery, options?: QueryOptions): Promise<QueryResult> => {
```

Comment:
Hm, I feel like that API could be cleaner. Maybe we should keep the string as first argument and add separate arguments for start/endTime?

Reply:
I think this approach is pretty standard. Frontend also does the same.

Reply:
Not super nice to use but alright